core/src/main/java/io/methvin/watcher/DirectoryWatcher.java

```diff
@@ -59,15 +59,24 @@
 
   // this is set to true/false depending on whether recursive watching is supported natively
   private Boolean fileTreeSupported = null;
+  private Boolean enableFileHashing = true;
 
   public static DirectoryWatcher create(Path path, DirectoryChangeListener listener) throws IOException {
-    return create(Collections.singletonList(path), listener);
+    return create(Collections.singletonList(path), listener, Boolean.TRUE);
+  }
+
+  public static DirectoryWatcher create(Path path, DirectoryChangeListener listener, Boolean enableFileHashing) throws IOException {
+    return create(Collections.singletonList(path), listener, enableFileHashing);
   }
 
   public static DirectoryWatcher create(List<Path> paths, DirectoryChangeListener listener) throws IOException {
+    return create(paths, listener, Boolean.TRUE);
+  }
+
+  public static DirectoryWatcher create(List<Path> paths, DirectoryChangeListener listener, Boolean enableFileHashing) throws IOException {
     boolean isMac = System.getProperty("os.name").toLowerCase().contains("mac");
     WatchService ws = isMac ? new MacOSXListeningWatchService() : FileSystems.getDefault().newWatchService();
-    return new DirectoryWatcher(paths, listener, ws);
+    return new DirectoryWatcher(paths, listener, ws, enableFileHashing);
   }
 
   public DirectoryWatcher(List<Path> paths, DirectoryChangeListener listener, WatchService watchService) throws IOException {
```

Comment:
This can delegate to the other constructor.

Reply:
Or maybe just omit it entirely, since the recommended way to create a `DirectoryWatcher` is through the `create` call. I'm not too worried about breaking API changes since the surface area is very small.

Reply:
I thought of removing it, but didn't want to brake the API. I will remove this constructor then.